src/codemagic/tools/app_store_connect.py

```diff
@@ -148,6 +154,72 @@ def _delete_resource(self, resource_manager, resource_id: ResourceId, ignore_not
             else:
                 raise AppStoreConnectError(str(api_error))
 
+    @cli.action('list-testflight-builds',
+                BuildArgument.APPLICATION_ID_OPTIONAL,
+                BuildArgument.EXPIRED,
+                BuildArgument.NOT_EXPIRED,
+                BuildArgument.BUILD_ID_OPTIONAL,
+                BuildArgument.PRE_RELEASE_VERSION,
+                BuildArgument.PROCESSING_STATE,
+                BuildArgument.BUILD_NUMBER,
+                BuildArgument.ORDERING,
+                BuildArgument.REVERSE)
+    def list_testflight_builds(self,
+                              application_id: Optional[Union[str, ResourceId]] = None,
```

Comment:
Type could be just `Optional[ResouceId]` if the argument is defined properly. The same applies to `build_id`.

Reply:
fixed [here](https://github.com/codemagic-ci-cd/cli-tools/pull/67/commits/71f2ca10806aa46be85d0feca84182780939f3d7#diff-c655fbe88a18ddd4d367405c06673842a69855a44563b37bd3b06625bf712419R166)